src/components/Contacts/ContactsMap/ContactsMapPanel.tsx

```diff
@@ -169,7 +169,9 @@ export const ContactsMapPanel: React.FC<ContactMapsPanelProps> = ({
     <>
       <Box padding={2}>
         <Box display="flex" justifyContent="space-between" alignItems="center">
-          <Typography variant="h6">{t('Partners by Status')}</Typography>
+          <Typography variant="h6" id="left-panel-header">
```

Comment:
Just want to make double check we are certain none of these left-panel-header ids can be on the screen at the same time.

Reply:
I just fixed the one place where `SidePanelsLayout` was nested inside itself resulting in duplicate ids.